sys/registry/namespace/sys/namespace_sys_board_led.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2023 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_registry_namespace_sys_board_led RIOT Registry Schema: BOARD_LED
+ * @ingroup     sys
+ * @brief       RIOT Registry BOARD_LED Schema representing the basic structure of a BOARD LED
+ * @{
+ *
+ * @file
+ *
+ * @author      Lasse Rosenow <lasse.rosenow@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+#include "kernel_defines.h"
+#include "registry.h"
+#include "registry/namespace/sys.h"
+
+#include "registry/namespace/sys/board_led.h"
+
+#if IS_USED(MODULE_REGISTRY_NAMESPACE_SYS_BOARD_LED) || IS_ACTIVE(DOXYGEN)
+
+/* Mapping */
+static void mapping(const registry_parameter_id_t parameter_id, const registry_instance_t *instance,
+                    void **val, size_t *val_len)
+{
+    registry_sys_board_led_instance_t *_instance =
+        (registry_sys_board_led_instance_t *)instance->data;
+
+    switch (parameter_id) {
+    case REGISTRY_SYS_BOARD_LED_ENABLED:
+        *val = &_instance->enabled;
+        *val_len = sizeof(_instance->enabled);
+        break;
+    }
+}
+
+/* Schema */
```

Comment:
Nit: could be considered as part of the schema but I was confused that it says schema here and the actual schema type is further down.

Reply:
It could not only be considered as part of the schema, it IS part of the schema.

But I will try to improve the comments to make it not confusing.